ui/src/components/yaml/styles.module.less

```diff
@@ -208,9 +208,7 @@
   .yaml_content_diagnosisPanel {
```

Comment:
Removing min-height and max-height could cause layout issues. Consider keeping these properties to ensure consistent sizing.